src/Bicep.Core/Emit/ExpressionEmitter.cs

```diff
@@ -314,6 +314,46 @@ public void EmitObjectProperties(ObjectSyntax objectSyntax, ISet<string>? proper
             }
         }
 
+        public void EmitModuleParameterValue(SyntaxBase syntax)
+        {
+            if (syntax is InstanceFunctionCallSyntax instanceFunctionCall && string.Equals(instanceFunctionCall.Name.IdentifierName, "getSecret", StringComparison.Ordinal))
```

Comment:
>StringComparison.Ordinal [](http://example.com/codeflow?start=146&length=24)

Should use `LanguageConstants.IdentifierComparison`. It's the same functionally but it makes it easy to find all the places where we compare identifiers.